scripts/fastsp_script/fsp_test.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+
+"""
+###remember sys.argv - can create variables using indexes with sys.argv!  
+    
+"""
+
+dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
```

Comment:
The goal **at the end** is to have  `overall_path = sys.argv[1] ## would be 'perturbed_alignments_Drosophila_AA' for example`. Then, list all the paths inside overall_path, and loop over those. What have here is a great start for initial development though!